02-technical-guides/hierarchies/01-hierarchy-fundamentals.ipynb

```diff
@@ -0,0 +1,1592 @@
+{
```

Comment:
<p>Minor point here: I wouldn't classify Order ID an 'inherently numerical column' as one can choose an ID convention that includes letters. Columns that are inherently numerical are things like "year" or "price." And here, the first would be better as a hierarchy.  I think this can be addressed by simply removing the word "inherently."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/890/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Mmmm very good point! Took out the word "inherently".</p>